routers/web/web.go

```diff
@@ -974,6 +974,8 @@ func registerRoutes(m *web.Router) {
 	// end "/repo": create, migrate, search
 
 	m.Group("/{username}/-", func() {
+		m.Post("/markup", web.Bind(structs.MarkupOption{}), misc.Markup)
```

Comment:
It should be placed under `/-/markup` because it doesn't really belong to "user"